examples/openai_embedding/requirements.txt

```diff
@@ -0,0 +1,4 @@
+openai
```

Comment:
Should we lock the version of each requirement?

Reply:
No, we need to keep tidb-vector updated in order to work with the latest versions of other packages.

Reply:
Roger that. That makes sense.